driver-legacy/src/main/com/mongodb/MongoClientOptions.java

```diff
@@ -99,8 +102,10 @@
 
     private final AutoEncryptionSettings autoEncryptionSettings;
     private final ServerApi serverApi;
+    private final Long timeout;
```

Comment:
MongoClientOptions uses a different variable name convention for timeouts - no `MS` suffix.